### PR TITLE
Fix issue with import from external source page

### LINKED ISSUE #997: search results not displayed while searching external sources
**Describe the bug**

When a search against external sources is performed in order to import data from them, results are found but not correctly displayed in the UI

**To Reproduce**
1. log in into DSpace
2. Go to "MyDspace" section
3. start a submission from an external source
4. select source and enter a query string
5. if results are found, no data are displayed

Schermata da 2021-01-25 18-49-20

**Expected behavior**
Result data returned by dspace rest correctly displayed

**Related work**
Link to any related tickets or PRs here.

### PR DESCRIPTION
**Pull request by atarix83**
## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #997 

## Instructions for Reviewers
Go to the the import external page and verify that the error reported in the issue no longer occurs

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [x] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [x] My PR doesn't introduce circular dependencies
- [x] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [x] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

File: src/app/core/data/comcol-data.service.spec.ts [hunk 1]
```diff
@@ -162,11 +162,12 @@ describe('ComColDataService', () => {
     });
 
     describe('if the scope Community can\'t be found', () => {
-      it('should throw an error', () => {
+      it('should throw an error', (done) => {
         const result = service.getBrowseEndpoint(options);
         const expected = cold('--#-', undefined, new Error(`The Community with scope ${scopeID} couldn't be retrieved`));
 
         expect(result).toBeObservable(expected);
+        done();
```
Comment (artlowel) [inline src/app/core/data/comcol-data.service.spec.ts:170]: I think this change is a bit strange. `done` should only be useful when it's used in a callback function for an async call. So I don't think it's doing anything here, in the main body of the `it` function. I think the result would be exactly the same if it was left out.

**Review by artlowel — APPROVED** [review]
Thanks @atarix83!

File: src/app/core/data/comcol-data.service.spec.ts (hunk 1 above)
Comment (atarix83) [reply src/app/core/data/comcol-data.service.spec.ts]: I removed the done and added a take(1) to observable to avoid the issue in the afterAll

**Comment by tdonohue** [comment]
Merging with two approvals.  This includes a small (hopeful) fix to tests in `comcol-data.service.spec.ts` to try to avoid seemingly random afterAll errors we are getting on some CI builds.  If those prove to not work entirely, we can always try again in a future PR.

**Pull request merged**